lib/include/mxl/time.h

```diff
@@ -17,59 +17,59 @@ extern "C"
 #define MXL_UNDEFINED_INDEX UINT64_MAX
 
     /**
-     * Get the current head index based on the current system TAI time
+     * Get the current ringbuffer index based on the current system TAI time
      * Index 0 is defined to be index at the beginning of the epoch as defined by SMPTE ST 2059
      *
      * \param[in] editRate The edit rate of the Flow
-     * \return The head index or MXL_UNDEFINED_INDEX if editRate is null or invalid
+     * \return The index or MXL_UNDEFINED_INDEX if editRate is null or invalid
      */
-MXL_EXPORT
-    uint64_t mxlGetCurrentHeadIndex(Rational const* editRate);
+    MXL_EXPORT
+    uint64_t mxlGetCurrentIndex(Rational const* editRate);
```

Comment:
For this specific function the word `Head` made sense, because it doesn't operate on arbitrary timestamps but the **current** time.

Reply:
I'm convinced.